webauthn-authenticator-rs/examples/authenticate/main.rs

```diff
@@ -51,7 +51,7 @@ impl From<UvPolicy> for UserVerificationPolicy {
 }
 
 fn select_transport<U: UiCallback>(ui: &U) -> impl AuthenticatorBackend + '_ {
-    let mut reader = AnyTransport::new().unwrap();
+    let mut reader = block_on(AnyTransport::new()).unwrap();
```

Comment:
Would it be worth making this fn async instead of the inner block_on?

Reply:
Done